src/pgzx/c.zig

```diff
@@ -4,6 +4,7 @@ const includes = @cImport({
     @cInclude("postgres.h");
     @cInclude("postgres_ext.h");
 
+    @cInclude("access/hash.h");
```

Comment:
I can't tell how this file is sorted, so I just put the new import at the top.

Reply:
I might have put it after `varatt.h` :). But feel free to keep it here.